plugins/labhub.py

```diff
@@ -301,6 +301,15 @@ def difficulty_level(user, iss):
                         return False
                 elif self.GH3_ORG.is_member(user):
                     return True
+            
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpswpgh65g/plugins/labhub.py
+++ b/tmp/tmpswpgh65g/plugins/labhub.py
@@ -301,7 +301,7 @@
                         return False
                 elif self.GH3_ORG.is_member(user):
                     return True
-            
+
             @register_check
             def block_gci_issue_assignment(user, iss):
                 """
```